openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -168,18 +164,55 @@ def add_to_transaction(neo4j_entities, transaction):
             transaction.create(entity)
 
     @staticmethod
-    def should_dump_course(course_key):
+    def get_command_last_run(course_key, graph):
+        """
+        This information is stored on the course node of a course in neo4j
+        :param course_key: a CourseKey
```

Comment:
We might want to have doc string which follows edx developer guide.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings